client/src/lib/normalize-widget-data.ts

```diff
@@ -48,13 +62,23 @@ function calculateMapTotal(mapData: WidgetMapData): number {
 }
 
 /**
- * Calculates percentage values for chart data using individual totals
+ * Calculates percentage values for chart data using the total value
+ * without N/A, plus filters out N/A from the array
  */
 function normalizeChartData(chartData: WidgetChartData): WidgetChartData {
-  return chartData.map((entry) => ({
-    ...entry,
-    value: calculatePercentage(entry.value, entry.total),
-  }));
+  const totalWithoutNA = chartData.reduce(
+    (acc, curr) => (curr.label !== "N/A" ? acc + curr.value : acc),
+    0,
+  );
```

Comment:
Shouldn't it be easier having a function that exclusively gets the total of N/A responses and then applying in different contexts? I think it could be used here and [here](https://github.com/Vizzuality/4-growth/pull/171/files#diff-293deeb9734550a9dc1457c6a978bd1191e18e261f201688e077ef0cd0567f4bR34-R44)